rskj-core/build.gradle

```diff
@@ -223,7 +218,10 @@ task javadocJar(type: Jar) {
 }
 
 jar {
+    def prevDuplicateStrategy = duplicatesStrategy
```

Comment:
could you please elaborate on why we need this trick?

Reply:
Fixed in last commit